docs/reference/admonitions.md

```diff
@@ -231,50 +231,52 @@ _Result_:
 :octicons-beaker-24: Experimental
 
 Admonitions and [Details][11] can also be rendered as inline blocks (i.e.
-sidebars), moving them to the right using the `inline` + `end` modifiers, or
-to the left using only the `inline` modifier.
+sidebars), placing them to the left using the `inline` modifier, or
+to the right with an additional context modifier: `inline end`.
 
-=== "inline end"
+Admonitions that use `inline` should be declared prior to the content block you want to place them beside.
+
+=== "inline"
 
     _Example_ / _Result_:
 
-    !!! info inline end
+    !!! info inline
```

Comment:
The blocks are swapped (`inline end` coming before `inline`) because in left-to-right languages, which English is one of, it's more common to place sidebar content to the right, as it doesn't interrupt the left vertical axis.

We can and should add the additional note on placement, though! 😊

Reply:
> because in left-to-right languages, which English is one of, it's more common to place sidebar content to the right

I understand that, but the placement is visualized in the example, and for English presenting left than right variants regardless of context is the usual expectation of order.

The switch was however as cited earlier for better implying the inline admonition must also be declared first, complimenting the text note added above it. If you want further clarity on left or right positioning, that can be better handled in the Tab names appending `(left-side)` or `(right-side)`?

Your docs though, I can remove the switch if you disagree with it being a UX improvement.